Include/cpython/code.h

```diff
@@ -224,9 +224,14 @@ PyAPI_FUNC(int) PyCode_Addr2Line(PyCodeObject *, int);
 
 PyAPI_FUNC(int) PyCode_Addr2Location(PyCodeObject *, int, int *, int *, int *, int *);
 
-typedef enum PyCodeEvent {
-  PY_CODE_EVENT_CREATE,
-  PY_CODE_EVENT_DESTROY
+#define PY_FOREACH_CODE_EVENT(V) \
+    V(CREATE)                 \
+    V(DESTROY)
+
+typedef enum {
+    #define DEF_EVENT(op) PY_CODE_EVENT_##op,
```

Comment:
Shouldn't this also have the PY_ prefix?

Reply:
We don't generally use them for macros that are undefined right after, but you're right, it's a fairly generic name and it's technically possible for something to define DEF_EVENT before including Python.h